cli/src/main/scala/filodb.cli/CsvImportExport.scala

```diff
@@ -62,7 +62,7 @@ trait CsvImportExport extends StrictLogging {
     }
 
     // There might still be rows left after the latest flush is done, so initiate another flush
-    val activeRows = client.ingestionStats(dataset, version).numRowsActive
+    val activeRows = client.ingestionStats(dataset, version).head.numRowsActive
```

Comment:
Will a call to .head ever throw an error? Maybe prefer headOption getOrElse a default or whatever makes sense.